assets/chat/js/chat.js

```diff
@@ -871,16 +867,16 @@ class Chat {
 
   censor(nick) {
     for (const message of this.mainwindow.messages) {
-      if (message.user?.username === nick) {
+      if (message.user?.username === nick.toLowerCase()) {
```

Comment:
Just to confirm, is this what you think solves the issues with some users not being censored?

Reply:
Yep, basically! When someone uses the bot to !mute/!ban people, it bans the lowercase version of the username, so the censor function can't find the right user (unless their username is already all lowercase ofc). So the usernames gotta be normalized for it to work properly.
![ksnip_20231127-191007](https://github.com/destinygg/chat-gui/assets/41237021/365eb84c-6cf5-47f6-9a5e-8eeddd88e911)